CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 - **CPU Scaler:** Adding e2e test for the cpu scaler ([#2441](https://github.com/kedacore/keda/pull/2441))
```

Comment:
Random thought - We should create some strictness to alphabetically sort and always put General at the top.

Reply:
Last time I sorted it before the release. But agree.